README.md

```diff
@@ -208,3 +208,10 @@ Past Sponsors:
 ## License
 
 Licensed under [MIT](./LICENSE).
+
+The `fastify` CI guarantees that the accepted dependencies licenses are:
+- MIT
+- ISC
+- BSD-3-Clause
+- BSD-2-Clause
+- Apache-2.0
```

Comment:
I would include some different text here. We do not imply that we won’t accept dependencies under a different license, but we’ll just vet them.

Note, we have no dependencies on Apache-2, so it should be removed. This is a whitelist of what we have right now, not what we would accept in the future. As such, it should be kept to the minimum.